prov/efa/src/rxr/rxr_op_entry.h

```diff
@@ -83,6 +83,12 @@ struct rxr_atomic_ex {
 	int comp_iov_count;
 };
 
+enum rxr_cuda_copy_method {
```

Comment:
I thought we were trying to trend away from using "rxr" and replace it with "efa"